docs/PyOp.md

```diff
@@ -0,0 +1,66 @@
+# Python Operator 
+## Introduction
+To facilitate Python coders on model developing, onnxruntime provides a way to invoke operators implemented in Python.
+To use the feature, model designer needs to:
+1. Define Python operator node with all required attributes;
+2. Implement a Python module of all referred operators in required format;
+3. Place the Python module into python sys path, then do referencing with onnxruntime.
+
+## Usage
+Currently, Python operator is only published via source under onnxruntime/core/language_interop_ops, developers need to compile with“-enable_language_interop_ops”and override existing onnxruntime binary. Meanwhile, please also copy onnxruntime_pywrapper.dll or libonnxruntime_pywrapper.so or libonnxruntime_pywrapper.dylib to the path where onnxruntime binary is placed.
```

Comment:
Can you write this step by step? Like step 1- do this. step 2 - do that. etc.

Reply:
Sure, adding steps.